release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -120,6 +120,12 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-scale"]
 === Scale
 
+[id="ocp-4-7-cnf-latency-test"]
+//RN for CNF-854
```

Comment:
No need to include this comment, since you have a JIRA task noted in the comment for the PR itself. :slightly_smiling_face: 

Other than that, looks great!

Reply:
@ahardin-rh I have removed the comment. I have also fixed the linking issue that was causing the build to break. PR is ready to merge